Automattic/woocommerce-payments
# Add local release package support for PHPStan

#### Changes proposed in this Pull Request
By default, QIT tests are running against a [published version](https://qit.woo.com/docs/cli/running-tests#testing-a-published-extension), but they can also run against the local build. The latter is beneficial when working on PHPStan reports local and checking the results against your local changes. This PR adds the new `npm run test:qit-phpstan-local` command which will run tests with the QIT CLI `--zip` flag.









#### Testing instructions

* run `npm run test:qit-phpstan` and check the plugin version in the test result, it should be the latest published version of WooPayments
* navigate to `woocommerce-payments.php` and change the `Version` header to be any other random version
* run `npm run test:qit-phpstan-local` and confirm that the test result refers the version you provided in the `woocommerce-payments.php` `Version` header

-------------------


**Post merge**
